src/client/dom-processor/client-dom-adapter.js

```diff
@@ -41,6 +41,17 @@ export default class ClientDomAdapter extends BaseDomAdapter {
         return false;
     }
 
+    hasOnsubmitHandler (el) {
+        var attrs = el.attributes;
+
```

Comment:
just use `el.hasAttr('onsubmit`).
It is more fastly then iterative way in 10 times